### PR TITLE
Create Slockit.mediawiki

### PR DESCRIPTION
**Pull request by joschicif**
Thanks for reviewing! (The short description is quite long, I know. If you have an idea of what to delete please let me know!)

P.s.: I reached out to Slock.it asking them to complete the sections with the missing information and facts.

### CODE REVIEW

File: Slockit.mediawiki [hunk 1]
```diff
@@ -0,0 +1,109 @@
+=Slock.it=
+
+* Company Name: Slock.it GmbH<ref name=„website“>https://slock.it/, accessed on 27.06.2018.</ref>
+* Website: https://slock.it/
+* Founded: 2015<ref name=„website“/>
+* Stage: Proof of Concept<ref name=„iotlayer“>https://iot.slock.it/iotlayer.html, accessed on 27.06.2018.</ref>
+* Team Size: 20<ref name=„team“>https://slock.it/team.html, accessed on 27.06.2018.</ref>
+* Sector: Industrials (20)
+* Funding: $2M<ref name=„funding“>https://blog.slock.it/slock-it-secures-2-million-usd-seed-funding-to-build-next-generation-sharing-economy-platform-b795c6d1a92d, accessed on 28.06.2018.</ref>
+* Based on Blockchain: Ethereum main chain<ref name =„ethereum“>https://slock.it/ethereum_computer.html, accessed on 28.06.2018.</ref>, Ethereum Kovan Test Network<ref>https://slock.it/tc.html, accessed on 03.07.2018.</ref>
```
Comment (Chainomics) [inline Slockit.mediawiki]: Both? For different applications?

File: Slockit.mediawiki [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
...hereum_computer.html, accessed on 02.07.2018.</ref> Additionally, Slock.it started to develop another main product, the so-called IOT Layer. This project aims to offer a comprehensive software framework for the gateway between physical IoT devices and the virtual blockchain. The IOT Layer offers payment channels (on- and off-chain), hub services, a marketplace for assets and access control to the connected IoT devices. Furthermore, the IOT Layer integrates IoT devices into existing IoT clouds and IoT environments. By empowering machines to send and receive transactions, it also enables Machine to Machine and Machine to Human interactions.<ref name=„iotlayer“>https://iot.slock.it/iotlayer.html, accessed on 02.07.2018.</ref> In addition, the „Incubed“-client will be part of the IOT Layer offering the possibility to connect to multiple chains without storing any parts of the blockchain. The proposed solution works in a trustless environment. For this purpose, devices are connected to various nodes that are incentivized to provide correct information only to requests of the IoT device.<ref name=„blog1“>https://blog.slock.it/slock-it-iot-layer-f305601df963, accessed on 02.07.2018.</ref>
```
Comment (Chainomics) [inline Slockit.mediawiki]: German upper case G

File: Slockit.mediawiki [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+* Identification code: N/A
+* Token Type: N/A
+* Link: N/A
+
+
+==Funding==
+
+2017: Slock.it raised about $2M in a seed funding round.<ref name=„funding“/>
+
+Additional Information: In 2016, Slock.it created a Decetralized Autonomous Organization (DAO) called „The DAO“, a set of smart contracts operating on the Ethereum Classic blockchain. Investors can send coins to the DAO and later vote on how to allocate the collected funds among several project proposals. The idea of Slock.it was to make a proposal of their Universal Sharing Network/Ethereum Computer project to raise funds in this way. In the initial phase of The DAO, an attacker discovered a security vulnerability in the code and managed to withdraw around 3.5M ETH (~$50M). Consequently, the community decided to recover the lost funds with a hard fork of the blockchain that resulted in the independently operating Ethereum Blockchain (where „The DAO“-attack was made undone) and the Ethereum Classic Blockchain (where the code of „The DAO“ can still be found).<ref name=„historydao“>https://blog.slock.it/the-history-of-the-dao-and-lessons-learned-d06740f8cfa5, accessed on 02.07.2018.</ref>
```
Comment (Chainomics) [inline Slockit.mediawiki]: ...smart contracts operating on the Ethereum blockchain.

...an attacker discovered a security flaw... or
...an attacker discovered a vulnerability...

The hack lead to an ideological dissent in the community, with some people arguing in favor of the immutability of the ledger and others in favor of the invalidation of the hack. The disagreement lead to change in the consensus rules and consequently to a forking event (Ethereum and Ethereum Classic).

File: Slockit.mediawiki (hunk 2 above)
Comment (Chainomics) [reply Slockit.mediawiki]: Maybe add an examples of car sharing or airb'nb like services.

File: Slockit.mediawiki (hunk 1 above)
Comment (joschicif) [reply Slockit.mediawiki]: They launched an application for iOS and Android lately running on the Ethereum Kovan Test Network at the moment. However, their Ethereum Computer project will connect IoT-devices to the Ethereum blockchain (main chain). Would you just mention one of the two?

Comment (Chainomics) [reply Slockit.mediawiki]: No, that's fine. If you still have space in the short description mention it there.

File: Slockit.mediawiki (hunk 3 above)
Comment (Chainomics) [reply Slockit.mediawiki]: Look at it as an inspiration :) If you decide to include it, it is meant to replace anything behind consequently. But it's fine if you combine the two.

**Comment by joschicif** [comment]
Some minor changes regarding summary and references due to an update of Slock.it's website. Furthermore, I included Jork Leonhardt's (Head of Operations of Slock.it) additions and suggestions.

@Chainomics: I also included your suggestions and added a brief example to the short description.

Question: 
Jork stated that Slock.it's revenue model is/will be as follows:
* Slock.it IoT Layer: Dual License 
* Universal Sharing Network: Revenue Participation

Do you think I classified it correctly?

**Comment by Chainomics** [comment]
Great changes!

**Comment by Chainomics** [comment]
@Rothj please review

**Pull request merged**